Makefile

```diff
@@ -55,13 +55,10 @@ latexpdf:
 	cp $(BUILD_DIR)/latex/*.pdf $(BUILD_DIR)/html/
 	@echo "Build finished."
 
-l10n:
+gettext:
 	@echo "Generating translatable files..."
-	$(SPHINX_BUILD) -c $(CONFIG_DIR) -b gettext $(SOURCE_DIR) $(BUILD_DIR)/gettext
+	$(SPHINX_BUILD) -c $(CONFIG_DIR) -b gettext $(SOURCE_DIR) locale/sources
 	@echo "Generation finished."
-	@echo "Localizing translation strings..."
-	sphinx-intl update -p $(BUILD_DIR)/gettext -l $(L10N_LANGUAGES)
-	@echo "Localization finished."
```

Comment:
Is that because the files are getting resolved / fetched / updated from transifex straight from the pots, so we don't need to generate the PO files by hand?

Reply:
exactly, we do a `tx pull` every week to get the latest translations, based on the `.tx/config` file